src/providers/test-submission/__tests__/test-submission.spec.ts

```diff
@@ -57,12 +58,13 @@ describe('TestSubmissionProvider', () => {
         index: 0,
         slotId: '',
         payload: {},
+        status: TestStatus.Completed,
       }] as TestToSubmit[]).subscribe();
 
-      httpMock.expectOne('https://www.example.com/api/v1/test-result');
+      httpMock.expectOne('https://www.example.com/api/v1/test-result?partial=false');
 
```

Comment:
I don't think we need `partial=false` for a full submission, see https://wiki.dvsacloud.uk/display/MES/Autosave+LLD